addons/binding/org.openhab.binding.gpstracker/ESH-INF/i18n/gpstracker_en.properties

```diff
@@ -0,0 +1,25 @@
+# binding
+binding.gpstracker.name = GPSTracker Binding
+binding.gpstracker.description = GPSTracker binding with support of OwnTracks and GPSLogger trackers over HTTP 
+
+# thing types
+thing-type.gpstracker.tracker.label = GPS Tracker
+thing-type.gpstracker.tracker.description = Tracker application running on mobile device
+
+# channel types
+channel-type.gpstracker.lastReport.label = Last report timestamp
+channel-type.gpstracker.lastReport.description = Timestamp of the last report received from tracker
+channel-type.gpstracker.location.label = Tracker location
+channel-type.gpstracker.location.description = Location sent by the tracker
+channel-type.gpstracker.batteryLevel.label = Battery level
+channel-type.gpstracker.batteryLevel.description = Battery level of the device running the tracker application
+channel-type.gpstracker.regionPresence.label = Presence switch
+channel-type.gpstracker.regionPresence.description = Switch indicating region presence
+channel-type.gpstracker.regionEnterTrigger.label = Region enter
+channel-type.gpstracker.regionEnterTrigger.description = Fired when tracker enters a region with the region name as payload
+channel-type.gpstracker.regionLeave.label = Region leave
+channel-type.gpstracker.regionLeaveTrigger.description = Fired when tracker leaves a region with the region name as payload
+
+#dynamic
+channel.gpstracker.presence.label=Tracker presence at {0}
+channel.gpstracker.distance.label=Current distance from {0}
```

Comment:
Capitalize it and keep it short: "Distance from {0}"

Reply:
OK